### PR TITLE
Jw community2

### PR DESCRIPTION
**Pull request by jeressia**
styled the resolved tag on community page

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,3599 @@
 {
   "name": "safespace",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (drteresavasquez) [inline package-lock.json:4]: This may break the app. Was this the branch that you installed the dependency on?

File: yarn.lock [hunk 2]
```diff
@@ -91,63 +55,18 @@
...
-  "version" "7.16.0"
-  dependencies:
-    "@babel/types" "^7.16.0"
-
-"@babel/helper-module-imports@^7.0.0", "@babel/helper-module-imports@^7.15.4", "@babel/helper-module-imports@^7.16.0":
+"@babel/helper-module-imports@^7.0.0", "@babel/helper-module-imports@^7.15.4":
   "integrity" "sha512-kkH7sWzKPq0xt3H1n+ghb4xEMP8k0U7XV3kkB+ZGy69kDk2ySFW1qPi06sjKzFY3t1j6XbJSqr4mF9L7CYVyhg=="
   "resolved" "https://registry.npmjs.org/@babel/helper-module-imports/-/helper-module-imports-7.16.0.tgz"
   "version" "7.16.0"
   dependencies:
```
Comment (drteresavasquez) [inline yarn.lock:62]: Same for this file. We may need to gitignore it because we are passing issues around. We need to remove this from this commit.

**Review by drteresavasquez — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 2 above)
Comment (jeressia) [reply yarn.lock:62]: Should I gitignore package-lock and yarn lock or just remove these lines?

**Review by jameka23 — COMMENTED** [review]
Screen Shot 2021-11-16 at 3 36 23 PM
is this suppose to happen @jeressia with the two tags? just curious

**Review by jameka23 — APPROVED** [review]
👍

**Comment by jeressia** [comment]
@jameka23 It's not supposed to happen but it's because some posts had null in the tags. There is another ticket that addresses that by Dr. T.

**Review by jameka23 — COMMENTED** [review]
yeah I just looked at that PR and the tags are straight on that PR @jeressia

**Review by drteresavasquez — APPROVED** [review]
👍🏼

**Pull request merged**